src/storage/storage.ts

```diff
@@ -862,4 +862,13 @@ export class Storage implements IStorage {
   async getGapLimit(): Promise<number> {
     return this.store.getGapLimit();
   }
+
+  /**
+   * Return if the loaded wallet was started from a hardware wallet.
+   * @returns {Promise<boolean>}
+   */
+  async isHardwareWallet(): Promise<boolean> {
```

Comment:
Does it make sense to create a method in both facades like this?

```
async isHardwareWallet() {
  return this.storage.isHardwareWallet();
}
```

If we don't do that, the wallets would need to access a storage method, which I prefer to avoid. What do you think?

Reply:
Done